### DISCUSSION TITLE
Sharing API communication - will hyundai_kia_connect_api be a first class citizen?

### DISCUSSION THREAD

**Discussion by ragdk** [Hyundai-Kia-Connect/kia_uvo#341] [Q&A] [answered]
Hi All,

In the repo https://github.com/Hyundai-Kia-Connect/hyundai_kia_connect_api @fuatakgun state that this HACS integration is the primary user of hyundai_kia_connect_api. However, it seems this repo does not make use of the hyundai_kia_connect_api package, and thus the hyundai_kia_connect_api package is stale? 

I think your initial intentions with hyundai_kia_connect_api is great - i.e. a separate python package for the Uvo/Bluelink communication, which can be used in other projects besides Home Assistant. This would also open up for more people to contribute.
How are your thoughts on that?

ping @fuatakgun and @cdnninja

Best,
Rasmus

**Comment by cdnninja** [comment, marked as answer]
We ran out of steam but yes.  Version 2.0 of Kia UVO exists in a branch called external api here that consumes that api. 

It's almost a full re write and not feature parity which is why it hasn't merged.  It's close though.
